app/src/main/java/com/dodo/flashcards/util/Screen.kt

```diff
@@ -0,0 +1,29 @@
+package com.dodo.flashcards.util
+
+sealed class Screen(val route: String) {
+    companion object{
+
+    }
+
+    object Login : Screen("Login")
+    object Register : Screen("Register")
+    object Welcome : Screen("Welcome")
+
+    object Categories : Screen("Category")
+    object Decks : Screen("Decks")
+    object CreateCategory : Screen("AddCategory")
```

Comment:
Nit, consider removing extra line in the middle and ordering these alphabetically